plugins/reports/api/reports.js

```diff
@@ -45,6 +45,11 @@ var metrics = {
     "events": {},
     "views": {}
 };
+var metricProps = {
+    "analytics": ["metric", "count", "change"],
+    "events": ["event", "count", "change"],
```

Comment:
If it is hard, then not now, but eventually I know clients will ask us to add support for sum and dur properties of events too

Reply:
yes, maybe we need to start with a new ticket because now it applies in the default 3 columns table template,  otherwise we need to separate table rendering logic, that need more time to achieve.